axelor-supplychain/src/main/java/com/axelor/apps/supplychain/service/workflow/WorkflowVentilationServiceSupplychainImpl.java

```diff
@@ -254,14 +254,18 @@ private void stockMoveProcess(Invoice invoice) throws AxelorException {
     // update qty invoiced in stock move line
     for (InvoiceLine invoiceLine : invoice.getInvoiceLineList()) {
       StockMoveLine stockMoveLine = invoiceLine.getStockMoveLine();
+      StockMove stockMove = stockMoveLine.getStockMove();
```

Comment:
It will be better if we add this line inside if block after null check of stockMoveLine